src/content/popup/kanji.ts

```diff
@@ -344,80 +343,14 @@ function renderReferences(
   entry: KanjiResult,
   options: PopupOptions
 ): HTMLElement {
-  const referenceTable = html('div', {
-    class: 'references',
-    lang: getLangTag(),
-  });
-
-  const referenceNames = getSelectedReferenceLabels(
-    options.kanjiReferences,
-    browser.i18n.getMessage.bind(browser.i18n)
+  const containerElement = html('div', { lang: getLangTag() }); // temp solution, remove container when containing element is reactified.
```

Comment:
Not sure we need the lang tag on this container element? I think the table sets its own lang tags?

Reply:
Yeah, some of this was cruft left over from changes I made during the conversion.